sbin/cubeit

```diff
@@ -197,8 +197,21 @@ fi
 
 ## Load configuration file(s)
 if [ -z ${CONFIG_FILES} ]; then
```

Comment:
We should have a flag that forces interactive mode, rather than there simply not being any config files.

That way interactive mode could be seeded with settings from --config if needed.